server/model/monitor.js

```diff
@@ -48,14 +48,11 @@ class Monitor extends BeanModel {
         let obj = {
             id: this.id,
             name: this.name,
-            sendUrl: this.sendUrl,
+            sendUrl: !!this.sendUrl,
+            url: this.url,
```

Comment:
why did you move `url` up here?

There are cases where we don't want to expose this information
=> currently, this would leak that as far as I can see..

Reply:
@CommanderStorm 
It's used in `showLink()`. Should I have not deleted the following after all?https://github.com/louislam/uptime-kuma/blob/46d8744fa40cf6a5773578252af655058437b1e6/server/model/monitor.js#L55-L57

Leaving this if-statement causes the issue described in #3794 because when `sendUrl` is false, the `url` isn’t sent, and the result of `showLink()` also becomes false.